app/src/org/commcare/utils/CachingAsyncImageLoader.java

```diff
@@ -44,14 +44,6 @@ public void display(String url, ImageView imageView, int defaultResource,
         }
     }
 
-    @SuppressLint("NewApi")
-    private class TCLruCache extends LruCache<String, Bitmap> {
-
-        public TCLruCache(int maxSize) {
-            super(maxSize);
-        }
-    }
-
```

Comment:
Pretty sure there is no need for this class. Making this nested class non-static means it can potentially leak memory (I don't think it does in this case, but worth removing so we don't have to worry about it)